README.md

```diff
@@ -1,15 +1,16 @@
+# Notice: this project will be deprecated and is succeeded by [videojs-http-streaming](https://github.com/videojs/http-streaming). VHS supports HLS and DASH and is built into video.js 7, see the [video.js 7 blog post](https://blog.videojs.com/video-js-7-is-here/)
+
 # video.js HLS Source Handler
 
 [![Build Status][travis-icon]][travis-link]
 [![Slack Status][slack-icon]][slack-link]
 [![Greenkeeper badge][greenkeeper-icon]][greenkeeper-link]
 
-
 Play back HLS with video.js, even where it's not natively supported.
 
 Lead Maintainer: Jon-Carlos Rivera [@imbcmdth](https://github.com/imbcmdth)
 
-Maintenance Status: Stable
+Maintenance Status: Deprecated
```

Comment:
We may want to wait to update this line until it's officially deprecated.

Reply:
we can leave this as deprecated but then archive the repo when it's actually deprecated